.gitlab-ci.yml

```diff
@@ -226,626 +226,12 @@ default:
       vault:                       cicd/gitlab/$CI_PROJECT_PATH/VALIDATOR_KEYS_CHAOS@kv
       file:                        false
 
-#### stage:                       .pre
```

Comment:
Those changes will be reverted before the PR is merged. I've removed the other steps other than `check-dependent-projects` so that the pipeline is fast during development.